staging/src/k8s.io/pod-security-admission/api/helpers.go

```diff
@@ -144,12 +144,59 @@ func (lv LevelVersion) String() string {
 	return fmt.Sprintf("%s:%s", lv.Level, lv.Version)
 }
 
+// Determines whether two LevelVersions are functionally equivalent. LevelVersions are considered
+// equivalent if both are privileged, or both levels & versions are equal.
+func (lv *LevelVersion) Equals(other *LevelVersion) bool {
+	return (lv.Level == LevelPrivileged && other.Level == LevelPrivileged) ||
+		(lv.Level == other.Level && lv.Version == other.Version)
+}
+
 type Policy struct {
 	Enforce LevelVersion
 	Audit   LevelVersion
 	Warn    LevelVersion
 }
 
+func (p *Policy) String() string {
+	return fmt.Sprintf("{enforce=%#v audit=%#v warn=%#v}", p.Enforce, p.Audit, p.Warn)
```

Comment:
I like the brackets because it makes it clear that the 3 components are grouped as a single policy, not 3 separate things. Although I guess it doesn't matter in this case... I think I've just spent too much time staring at code.

BTW, the format used here is closer to the go format you get with `%#v` than json.

Reply:
Without the brackets, this reads a little weirdly in the full warning context:

```
namespace test is exempt from Pod Security, and the policy enforce=restricted:v1.0, audit=restricted:latest, warn=baseline:latest will be ignored
```

calling these `policies` might help, but then we need code to distinguish between the singular & plural cases.

WDYT?

Reply:
then Policy#CompactString can just return the policy bits, and the caller can put it in parens

Reply:
Done.